src/lib/requests/getPublicAccounts/index.ts

```diff
@@ -32,30 +32,22 @@ export const getPublicAccounts = async (
   )
     throw new Error(errors.onlyOneRangeValue);
 
-  if (
-    options &&
-    (options.rangeStart || options.rangeStart === 0) &&
-    options.rangeEnd
-  ) {
-    const { data: extended_user_profiles, error } = await supabase
-      .from<definitions["extended_user_profiles"]>("extended_user_profiles")
-      .select("*")
-      .range(options.rangeStart, options.rangeEnd);
-
-    if (error) throw error;
-    if (!extended_user_profiles) return [];
-
-    const accounts = extended_user_profiles.map(mapPublicAccount);
-    return accounts;
-  } else {
-    const { data: extended_user_profiles, error } = await supabase
-      .from<definitions["extended_user_profiles"]>("extended_user_profiles")
-      .select("*");
-
-    if (error) throw error;
-    if (!extended_user_profiles) return [];
-
-    const accounts = extended_user_profiles.map(mapPublicAccount);
-    return accounts;
-  }
+  const defaultRangeStart = 0;
+  const defaultRangeEnd =
+    Number.parseInt(process.env.NEXT_PUBLIC_SUPABASE_MAX_ROWS as string) ||
+    1000;
```

Comment:
Aren't we on 3000 by default?

Reply:
3000 is the amount of records the line chart can render. We are actually on `process.env.NEXT_PUBLIC_SUPABASE_MAX_ROWS`. The 1000 is just there as a fallback because _theoretically_ `NEXT_PUBLIC_SUPABASE_MAX_ROWS` can be undefined.

Reply:
👍🏽 
I normally create a module that does the checking of process.env. values for me
```ts
// env.ts
const NEXT_THING = process.env.NEXT_THING;
if(NEXT_THING === undefined) throw new Error('next thing is undefined')
export NEXT_THING
```

This is a little more work but it makes sure the values exist and I have a central place from where I import env vars. If I miss them I get an error.

Reply:
Also I don't scatter that many `parseInt (...) || 1` around the code

Reply:
But it is up to you.

Reply:
> I normally create a module that does the checking of process.env. values for me

That's a really good idea. Though I wouldn't make this change in this PR because we have a rather time-critical moment.

But I'll definitely consider this for the future!